src/routes/safe/store/actions/transactions/__tests__/utils/transactionHelpers.test.ts

```diff
@@ -0,0 +1,875 @@
+import { getMockedSafeInstance, getMockedTxServiceModel } from 'src/test/utils/safeHelper'
+
+import { makeTransaction } from 'src/logic/safe/store/models/transaction'
+import { TransactionStatus, TransactionTypes } from 'src/logic/safe/store/models/types/transaction'
+import makeSafe from 'src/logic/safe/store/models/safe'
+import { List, Map, Record } from 'immutable'
+import { makeToken, TokenProps } from 'src/logic/tokens/store/model/token'
+import { EMPTY_DATA } from 'src/logic/wallets/ethTransactions'
+import { ZERO_ADDRESS } from 'src/logic/wallets/ethAddresses'
+import {
+  buildTx,
+  calculateTransactionStatus,
+  calculateTransactionType,
+  generateSafeTxHash,
+  getRefundParams,
+  isCancelTransaction,
+  isCustomTransaction,
+  isInnerTransaction,
+  isModifySettingsTransaction,
+  isMultiSendTransaction,
+  isOutgoingTransaction,
+  isPendingTransaction,
+  isUpgradeTransaction,
+} from 'src/logic/safe/store/actions/transactions/utils/transactionHelpers'
+import { getERC20DecimalsAndSymbol } from 'src/logic/tokens/utils/tokenHelpers'
+
+const safeAddress = '0xdfA693da0D16F5E7E78FdCBeDe8FC6eBEa44f1Cf'
+const safeAddress2 = '0x344B941b1aAE2e4Be73987212FC4741687Bf0503'
+describe('isInnerTransaction', () => {
+  it('It should return true if the transaction to is our given safeAddress and the txValue is 0', () => {
+    // given
+    const transaction = getMockedTxServiceModel({ to: safeAddress, value: '0' })
+
+    // when
+    const result = isInnerTransaction(transaction, safeAddress)
+
+    // then
+    expect(result).toBe(true)
+  })
+  it('It should return false if the transaction to is our given safeAddress and the txValue is >0', () => {
+    // given
+    const transaction = getMockedTxServiceModel({ to: safeAddress, value: '100' })
+
+    // when
+    const result = isInnerTransaction(transaction, safeAddress)
+
+    // then
+    expect(result).toBe(false)
+  })
+  it('It should return false if the transaction to is not our given safeAddress', () => {
```

Comment:
This and previous test's descriptions:
> if the transaction to is our

Next tests' description:
>  if the transaction recipient is

Please stay consistent